x-pack/plugins/cross_cluster_replication/public/app/sections/home/follower_indices_list/components/context_menu/context_menu.js

```diff
@@ -146,18 +146,30 @@ export class ContextMenuUi extends PureComponent {
           }
 
           { followerIndexNames.length === 1 && (
-            <EuiContextMenuItem
-              icon="pencil"
-              onClick={() => this.editFollowerIndex(followerIndexNames[0])}
-            >
-              <FormattedMessage
-                id="xpack.crossClusterReplication.followerIndex.contextMenu.editLabel"
-                defaultMessage="Edit follower index"
-              />
-            </EuiContextMenuItem>
+            <Fragment>
+              <EuiContextMenuItem
+                icon="pencil"
+                onClick={() => this.editFollowerIndex(followerIndexNames[0])}
+              >
+                <FormattedMessage
+                  id="xpack.crossClusterReplication.followerIndex.contextMenu.editLabel"
+                  defaultMessage="Edit follower index"
+                />
+              </EuiContextMenuItem>
+
+              <EuiContextMenuItem
+                icon="inspect"
+                href={getIndexListUri(`name:${followerIndexNames[0]}`)}
+              >
+                <FormattedMessage
+                  id="xpack.crossClusterReplication.followerIndex.contextMenu.viewIndexManagementLabel"
+                  defaultMessage="View in Index Management"
```

Comment:
What do you think of moving this into the form row actions menu and removing it from the "Manage" button in both the table and the detail panel? It seems awkward for a user to select a single row just to try to view it index management. I think it'd more natural to expect it to appear when looking inside the row's actions. Also, the word "Manage" implies you're doing something to the follower index, as opposed to viewing it in another app so it seems a bit incongruous.

![image](https://user-images.githubusercontent.com/1238659/51696342-22d0d400-1fba-11e9-82f6-f8ec3d81ceaa.png)

In the detail panel, it's also redundant since we already have a link for viewing the index in Index Management. If we wanted to make it pop out more, we could move it into the footer but we could explore that later when we're polishing the UI.

![image](https://user-images.githubusercontent.com/1238659/51696538-92df5a00-1fba-11e9-9d3e-4b8a353611f8.png)

Reply:
good idea for both cases!

Reply:
^ will implement in followup